NUUtils/NUUtilities.j

```diff
@@ -274,6 +274,23 @@ function _IPAddress(string, canBeNull, canBeAll)
     return success ? null : "Not a valid address";
 }
 
+function _splitAndValidateIPAddress(string)
+{
+    if (!string)
+        return;
+
+    var IPAddressArray = string.split(",");
+    var successArray =[];
```

Comment:
there is no need to keep track of all the validation, if any of the strings is invalid return "Invalid addressed" right away!